### DISCUSSION TITLE
Visibilidade do SITE para o cliente, durante a criação.

### DISCUSSION THREAD

**Discussion by ibkgit** [community/community#62696] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Estou criando um site, e não estou conseguindo deixá-lo visivel pelo GITHUB PAGES para o cliente, para que possamos interagir.
Ex :https://ibkgit.github.io/provencion/index.html

**Comment by lucmsilva651** [comment]
Opa, eu não sou do suporte oficial mas gostaria de ajudar. Veja se seu repositório **[Prevention](https://github.com/ibkgit/Prevencion)** está configurado para o GitHub Pages. Vá nas configurações de seu repositório na aba Settings (ou se estiver usando um tradutor, vá até Opções) e selecione a opção Pages. Lá terá alguma opção chamada "Branch", onde terá uma caixa escrita "None". Selecione lá sua branch "main" e clique em "Save". Espere uns instantes (uns 2 minutinhos) e seu site estará pronto.
Depois de terminado, o link será esse: <i><del>https://ibkgit.github.io/provencion/</del></i>

Edit: você digitou o link incorreto, o certo seria: **[https://ibkgit.github.io/Prevencion/](https://ibkgit.github.io/Prevencion/)**
Edit²: caso queira uma demostração, basta visitar meus repositórios no meu perfil.
